aspnetcore/blazor/server-side-blazor-best-practices.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Server Blazor app best practices
```

Comment:
This doc seems like a mix of stuff. Could we instead put the guidance in the relevant doc? For example, the "Harden JS interop" section would go in the JS interop doc, "Guard against multiple dispatches" would go in the components doc, etc.? @guardrex thoughts?

Reply:
Sure thing. I'll try to place the content into relevant topics on the new PR with the usual docs updates. I think I can get to this tomorrow after Pre9 mop-up work.